### PR TITLE
All tasks done and working.

### PR DESCRIPTION
**Pull request by opiotrowsky**
Author: @opiotrowsky

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @opiotrowsky

**Comment by ziobron** [comment]
⏱ 3 XP extra for @opiotrowsky for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @opiotrowsky

**Comment by ziobron** [comment]
⏱ 3 XP extra for @opiotrowsky for on-time delivery

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include <chrono>
+#include <thread>
+#include <string>
+#include <functional>
```
Comment (lukasz-kukulka) [inline homework/schedule/schedule.hpp:4]: Alphabetic would be better

**Comment by ziobron** [comment]
Thanks @lukasz-kukulka for Code Review.
🏅 1 XP granted

File: homework/schedule/schedule.hpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+void schedule(std::function<void(std::string, double)> func, std::chrono::seconds duration, std::string text, double value) {
+    std::this_thread::sleep_for(duration);
+    func(text, value);
+}
+
+/*  Bonus: zaimplementuj schedule jako funkcję szablonową. Wystarczy wtedy tylko jedna jej wersja, bez żadnych przeciążeń. */
+template<typename T, typename... V>
+void schedule(T func, std::chrono::seconds duration, V... values) {
+    std::this_thread::sleep_for(duration);
+    func(values...);
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:33]: use perfcet forwarding here. std::forward<Args>(args)...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**